frontend/src/queries/nodes/InsightViz/InsightViz.tsx

```diff
@@ -68,14 +69,14 @@ export function InsightViz({ uniqueKey, query, setQuery, context, readOnly }: In
     const isHorizontalAlways = useFeatureFlag('INSIGHT_HORIZONTAL_CONTROLS')
 
     const showIfFull = !!query.full
-    const disableHeader = !(query.showHeader ?? showIfFull)
-    const disableTable = !(query.showTable ?? showIfFull)
-    const disableCorrelationTable = !(query.showCorrelationTable ?? showIfFull)
-    const disableLastComputation = !(query.showLastComputation ?? showIfFull)
-    const disableLastComputationRefresh = !(query.showLastComputationRefresh ?? showIfFull)
+    const disableHeader = embedded || !(query.showHeader ?? showIfFull)
+    const disableTable = embedded || !(query.showTable ?? showIfFull)
+    const disableCorrelationTable = embedded || !(query.showCorrelationTable ?? showIfFull)
+    const disableLastComputation = embedded || !(query.showLastComputation ?? showIfFull)
+    const disableLastComputationRefresh = embedded || !(query.showLastComputationRefresh ?? showIfFull)
```

Comment:
We do have this `full: true/false` convention with various display nodes. Would it make sense to consider an "embedded" insight just one that forces `full` to be `false`?

Reply:
Sure, but how what would that mean for these conditions, since showIfFull is evaluated only as fallback?

Reply:
Ping